src/renderer/components/ProjectCreation/CloudConfigView.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+  Box,
+  IconButton,
+  Link,
+  Stack,
+  styled,
+  TextField,
+  Typography,
+} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import colors from 'renderer/colors';
+import { ChangeEvent, useState } from 'react';
+import useKeypress from 'renderer/utils/hooks';
+import { PrimaryButton, SecondaryButton } from '../Blocks/Buttons';
+import ipc from '../../ipc';
+
+const { openExternalLink } = ipc;
+
+interface Props {
+  prevView: () => void;
+  closeModal: () => void;
+  nextView: () => void;
+  projectName: string;
+  textToDisplay: string | null;
+}
+
+const CustomStack = styled(Stack)({ width: '100%' });
+
+const CustomColumnStack = styled(CustomStack)({ flexDirection: 'column' });
+
+const CustomRowStack = styled(CustomStack)({
+  flexDirection: 'row',
+  alignItems: 'center',
+});
+
+const Container = styled(Box)({
+  backgroundColor: colors.grey[700],
+});
+
+const CloudConfigView = ({
+  prevView,
+  closeModal,
+  nextView,
+  projectName,
+  textToDisplay,
+}: Props) => {
+  const [isAwaitingAPIKey, setAwaitingAPIKey] = useState<boolean>(true);
+  const [isAwaitingClientSecret, setAwaitingClientSecret] =
+    useState<boolean>(true);
+  const [APIKey, setAPIKey] = useState<string>('');
```

Comment:
`apiKey` and `setApiKey` - treating acronyms as lowercase words is the generally accepted way to use them in variable names (see https://stackoverflow.com/questions/15526107/acronyms-in-camelcase)